docker-compose.yml

```diff
@@ -10,14 +10,15 @@ services:
       context: .
       dockerfile: contrib/images/slinky.sidecar.e2e.Dockerfile
     volumes:
-      - ./config/local/oracle.json:/oracle/oracle.json
-      - ./config/local/market.json:/oracle/market.json
+      - ./config/dydx/oracle.json:/oracle/oracle.json
+      - ./config/dydx/market.json:/oracle/market.json
```

Comment:
What is the docker-compose used in again? e2e tests?

Reply:
nothing, just running locally